docs/source/contribution.rst

```diff
@@ -1,88 +1,66 @@
-Contributing guidelines
------------------------
+## Contributing guidelines
 
-Thank you for your interest in contributing to EvalAI! Here are a few
-pointers on how you can help.
+Thank you for your interest in contributing to EvalAI! Here are a few pointers about how you can help.
 
-Setting things up
-~~~~~~~~~~~~~~~~~
+### Setting things up
 
-To set up the development environment, follow the instructions in
-our README.
+To set up the development environment, follow the instructions in README.
```

Comment:
Remove readme and add EvalAI docs section where the installation instructions are provided.

Reply:
Done